Atomic/storage.py

```diff
@@ -171,6 +172,11 @@ def modify(self):
             if self.args.driver:
                 self._driver(self.args.driver)
             if self.args.rootfs and self.args.lvname:
+                if self.args.lvsize:
+                    cmd = 'bash -c ". /usr/lib/docker-storage-setup/libdss.sh; check_data_size_syntax {0}"'.format(self.args.lvsize)
+                    if os.system(cmd) != 0:
```

Comment:
Yeah it is happening. I just changed `except IOError` to `except subprocess.CalledProcessError` and it works. PTAL.